survey_form/index.css

```diff
@@ -1,124 +1,117 @@
-body{
-background-color: #66CDAA;
-font-family:'Noto Serif SC', serif;
-margin:0;
-}
-.navi{
-    width: 80%;
-    margin: 0 auto;
-}
-nav{
-    float: right;
-    display:flex;
-}
-header{
-    background: #F5F5F5;
-}
-header::after{
-    content: "";
-    display: table;
-    clear: both;
-}
-nav ul{
-    list-style: none;
-}
-nav li{
-    display: inline-block;
-    margin-left: 50px;
-}
-nav a{
-    text-decoration: none;
-    color: black;
-    font-family: 'Bangers';
-    text-transform: uppercase;
-    font-size: 25px;
-
-}
-nav a:hover{
-    display: inline-block;
-    text-align: center;
-    color:#66CDAA;
-    
-}
-.logo{
-    float: left; 
-    font-family: 'Bangers', cursive;
-    color: #66CDAA;
-    
-}
-#title{
-    color:black;
-    text-align: center;
-}
-#description{
-    text-align: center;
+body {
+  background-color: #66cdaa;
+  font-family: "Noto Serif SC", serif;
+  margin: 0;
+}
+.nav {
+  /* float: right; */
+  display: flex;
+  margin: 0 auto;
+}
+header {
+  background: #f5f5f5;
+}
+header::after {
+  content: "";
+  display: table;
+  clear: both;
+}
+.nav__list {
+  list-style: none;
+  float: right;
```

Comment:
Nie używa się już `float`ów - jest to dość stara technika, teraz możemy się bawić flexboxem i gridem :) 
Proponuję coś takiego:
```
.nav__list {
    display: flex; // żeby ułożyć elementy nawigacji "obok siebie"
    list-style: none; 
    padding: 0; // tag `ul` dostaje na start jakiś padding, który może wprowadzić tutaj zamieszanie, warto wyczyścić
}
```